tests/add_fee_amount.go

```diff
@@ -0,0 +1,103 @@
+package tests
+
+import (
+	"context"
+	"math/big"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ava-labs/teleporter/tests/network"
+	"github.com/ava-labs/teleporter/tests/utils"
+	localUtils "github.com/ava-labs/teleporter/tests/utils/local-network-utils"
+	"github.com/ethereum/go-ethereum/common"
+	. "github.com/onsi/gomega"
+)
+
+func AddFeeAmountGinkgo() {
+	AddFeeAmount(&network.LocalNetwork{})
+}
+
+func AddFeeAmount(network network.Network) {
+	subnets := network.GetSubnetsInfo()
+	subnetAInfo := subnets[0]
+	subnetBInfo := subnets[1]
+	teleporterContractAddress := network.GetTeleporterContractAddress()
+	fundedAddress, fundedKey := network.GetFundedAccountInfo()
+	ctx := context.Background()
+
+	// Use mock token as the fee token
+	mockTokenAddress, mockToken := localUtils.DeployExampleERC20(
+		context.Background(),
+		fundedKey,
+		subnetAInfo,
+	)
+	localUtils.ExampleERC20Approve(
+		ctx,
+		mockToken,
+		teleporterContractAddress,
+		big.NewInt(0).Mul(big.NewInt(1e18),
```

Comment:
Can this just be `big.NewInt(1e18)`? Unless there's a reason it needs to be larger, I think that's a cleaner option.

Reply:
fixed